test/infra/cdn-site-hosting/cdn-site-hosting-construct.test.ts

```diff
@@ -12,7 +12,9 @@ import { Template } from "@aws-cdk/assertions";
 
 // hosted-zone requires an environment be attached to the Stack
 const testEnv: Environment = {
-  region: "eu-west-1",
+  // region for Cloudfront Distribution certificates must be us-east-1
+  // https://stackoverflow.com/questions/37289994/aws-certificate-manager-do-regions-matter
+  region: "us-east-1",
```

Comment:
I think this is intended to represent the environment in which we are provisioning the entire stack, not just a Cloudfront Distribution certificate.

Let's say we wanted to add more resources to the construct that _are_ region-specific, so we have a mixture of things that are supposed to be statically `us-east-1` and things that are supposed to be provisioned in the specified region. If we use `us-east-1` as our test case, we would not be able to successfully test that our our region specifications are correct, since it'd be entirely possible that either:

1. We'd hard-coded everything to a single region, erroneously.
2. We'd made everything respect the specified region when some things _should_ be hard-coded to `us-east-1`.

For this reason, I'd be reluctant to hard-code to `us-east-1`. 

(To mitigate point 1, we should ultimately be testing to ensure that it respects different specified regions.)

Reply:
Your broader point about `us-east-1` would then involve hard-coding that correct region into line 20, IIUC.